tests/console/sysstat.pm

```diff
@@ -72,10 +72,14 @@ sub run {
     validate_script_output "mpstat",     sub { /CPU    %usr   %nice    %sys %iowait    %irq   %soft  %steal  %guest  %gnice   %idle/ };
     validate_script_output "sar -u",     sub { /CPU     %user     %nice   %system   %iowait    %steal     %idle/ };
     validate_script_output "sar -n DEV", sub { /IFACE   rxpck\/s   txpck\/s    rxkB\/s    txkB\/s   rxcmp\/s   txcmp\/s  rxmcst\/s   %ifutil/ };
-    if (is_sle('<=15-SP2') || is_leap('<=15.2')) {
-        validate_script_output "sar -b", sub { /tps      rtps      wtps   bread\/s   bwrtn\/s/ };
-    } else {
+    #from version 12.1.2 iostat supports discard I/O statistics.
+    #numify both version variables
+    my $installed_ver = sprintf("%3d%03d%03d", split('\.', script_output('rpm --qf "%{VERSION}\n" -q sysstat'))) + 0;
+    my $ref_ver       = sprintf("%3d%03d%03d", 12, 1, 2) + 0;
+    if ($installed_ver >= $ref_ver) {
```

Comment:
IMO we need to keep in mind that our "system under test" is not some certain app but whole ISO . your changing switching focus of testing from ISO to certain app. Sorry I don't have full picture in which SLE/Leap versions iostat was already upgraded to `12.1.2` and where it is still lower. I can imagine that reason why you choose this way is because `is_sle`/`is_leap` would look much uglier but nevertheless I would prefer it

Reply:
but couldn't it happen that version 12.1.2 ends up in 15.2 as well?

Reply:
yes it could and I assume **will** happen . and I totally agree that from algorithmic perspective smarter and better use version of app here . I am just saying that writing  beautiful and shiny code is not our main goal. First of all this code should fulfill it's main purpose . And it's main purpose is to test **ISO's** , and IMO with this PR we loosing it.

Reply:
Currently, there is 12.4.0 in [Factory](https://build.opensuse.org/package/show/openSUSE:Factory/sysstat) and in [tw](https://openqa.opensuse.org/tests/1377386#step/sysstat/3). Not sure, why there is such a gap between TW and others.
It is correct that our scope is above the package level testing, however I found it cumbersome to use `is_$distro` when the feature has been introduced in main stream package.

Reply:
If there is no huge gap between different distri versions, then it makes sense to leave as it was, but if we are going to make long condition expressions for each OS, then I prefer this way, which is testing a feature present in a specific version of the package.